package.json

```diff
@@ -28,9 +28,6 @@
     "email": "ismaelga@gmail.com"
   },
   "main": "dist/react-json-editor.js",
-  "peerDependencies": {
-    "react": ">=0.12.2 <1.0.0"
-  },
```

Comment:
@jahed Thanks for your reply. I will indeed do what you and @ismaelga suggested and create a new PR. For my interest, what did you end up replacing it with?
@ismaelga Are you yourself still dependent on this project?